tests/test_db_containers.py

```diff
@@ -36,18 +37,42 @@ def test_docker_run_mariadb():
             assert row[0] == '10.2.9-MariaDB-10.2.9+maria~jessie'
 
 
-@pytest.mark.skip(reason="needs oracle client libraries unavailable on Travis")
```

Comment:
Why is it commented?

Reply:
I removed the comment in the last commit